homeassistant/helpers/config_validation.py

```diff
@@ -1027,6 +1041,7 @@ def determine_script_action(action: dict) -> str:
     SCRIPT_ACTION_CHECK_CONDITION: CONDITION_SCHEMA,
     SCRIPT_ACTION_DEVICE_AUTOMATION: DEVICE_ACTION_SCHEMA,
     SCRIPT_ACTION_ACTIVATE_SCENE: _SCRIPT_SCENE_SCHEMA,
+    "branch": BRANCH_SCHEMA,
```

Comment:
I think you should use a constant instead of `"branch"`. It appears multiple times in the code.

Reply:
Yes, of course. There's lots of cleanup to be done once the idea is mature enough.